### PR TITLE
integrate money into expense. fix checkstyle errors

### PR DESCRIPTION
**Pull request by mukund1403**

### CODE REVIEW

**Comment by hafizuddin-a** [comment]
Thank you for updating storage to allow storing multiple currencies

File: src/main/java/seedu/duke/Balance.java [hunk 1]
```diff
@@ -35,12 +35,14 @@ public Map<String, Float> getBalanceList() {
 
     private void addExpense(Expense expense) {
         String payerName = expense.getPayerName();
-        List<Pair<String, Float>> payees = expense.getPayees();
+        List<Pair<String, Money>> payees = expense.getPayees();
```
Comment (Cohii2) [inline src/main/java/seedu/duke/Balance.java:38]: Noted change to List entry - will edit balanceList methods to parse list properly.

File: src/main/java/seedu/duke/Expense.java [hunk 2]
```diff
@@ -25,11 +26,11 @@ public class Expense {
      * @param description : Description of the expense
      * @param totalAmount : The total amount before being divided
      * @param payees      : ArrayList of pairs containing names of people who are involved in the transaction and
-     *                    the amount they owe (Index 0 is the payer and will also be added to the payees but as last index)
+     *                    the amount they owe
+     *                    (Index 0 is the payer and will also be added to the payees but as last index)
```
Comment (Cohii2) [inline src/main/java/seedu/duke/Expense.java:30]: Good use of comments to explain functionality.

File: src/main/java/seedu/duke/Group.java [hunk 3]
```diff
@@ -318,7 +318,7 @@ private void updateBalancesAfterSettlement(String payerName, String payeeName) {
             expense.clear();
         }
 
-        if (payerExpense.size() != 0) {
+        if (!payerExpense.isEmpty()) {
```
Comment (Cohii2) [inline src/main/java/seedu/duke/Group.java:321]: Good change to isEmpty() method

**Review by Cohii2 — COMMENTED** [review]
Noted changes to Expense class for use with Money class.

**Pull request merged**